tests/unit/app/query_test.py

```diff
@@ -59,12 +59,15 @@ def test_basic_operations(self):
         host2['service'] = host2_service
         self.assertEqual(host2, host2_get)
 
-        secondary_query = sorted(query.query_secondary_index(host1['service_repo_name']))
+        secondary_query = sorted(
+            query.query_secondary_index(host1['service_repo_name']),
+            key=lambda dct: sorted(dct.items()),
+        )
         self.assertEqual([host1, host2], secondary_query)
 
         query.delete(host2['service'], host2['ip_address'])
 
-        secondary_query = sorted(query.query_secondary_index(host1['service_repo_name']))
+        secondary_query = list(query.query_secondary_index(host1['service_repo_name']))
```

Comment:
sort is not needed? ah I see it was getting sorted twice

Reply:
there's only one element so sorting does nothing